opentsdb/getmetrics_opentsdb_oldapi.py

```diff
@@ -0,0 +1,298 @@
+    parser.add_option("-c", "--chunkSize",
+                      action="store", dest="chunkSize", help="Metrics per chunk")
+    (options, args) = parser.parse_args()
+
+    parameters = {}
+    if options.homepath is None:
+        parameters['homepath'] = os.getcwd()
+    else:
+        parameters['homepath'] = options.homepath
+    if options.serverUrl == None:
+        parameters['serverUrl'] = 'https://app.insightfinder.com'
+    else:
+        parameters['serverUrl'] = options.serverUrl
+    if options.chunkSize is None:
+        parameters['chunkSize'] = 50
+    else:
+        parameters['chunkSize'] = int(options.chunkSize)
+    return parameters
+
+
+def getAgentConfigVars():
+    configVars = {}
+    with open(os.path.join(parameters['homepath'], ".agent.bashrc"), 'r') as configFile:
+        fileContent = configFile.readlines()
+        if len(fileContent) < 6:
+            logger.error("Agent not correctly configured. Check .agent.bashrc file.")
+            sys.exit(1)
+        # get license key
+        licenseKeyLine = fileContent[0].split(" ")
+        if len(licenseKeyLine) != 2:
+            logger.error("Agent not correctly configured(license key). Check .agent.bashrc file.")
+            sys.exit(1)
+        configVars['licenseKey'] = licenseKeyLine[1].split("=")[1].strip()
+        # get project name
+        projectNameLine = fileContent[1].split(" ")
+        if len(projectNameLine) != 2:
+            logger.error("Agent not correctly configured(project name). Check .agent.bashrc file.")
+            sys.exit(1)
+        configVars['projectName'] = projectNameLine[1].split("=")[1].strip()
+        # get username
+        userNameLine = fileContent[2].split(" ")
+        if len(userNameLine) != 2:
+            logger.error("Agent not correctly configured(username). Check .agent.bashrc file.")
+            sys.exit(1)
+        configVars['userName'] = userNameLine[1].split("=")[1].strip()
+        # get sampling interval
+        samplingIntervalLine = fileContent[4].split(" ")
+        if len(samplingIntervalLine) != 2:
+            logger.error("Agent not correctly configured(sampling interval). Check .agent.bashrc file.")
+            sys.exit(1)
+        configVars['samplingInterval'] = samplingIntervalLine[1].split("=")[1].strip()
+    return configVars
+
+
+def getReportingConfigVars():
+    reportingConfigVars = {}
+    with open(os.path.join(parameters['homepath'], "reporting_config.json"), 'r') as f:
+        config = json.load(f)
+    reporting_interval_string = config['reporting_interval']
+    is_second_reporting = False
+    if reporting_interval_string[-1:] == 's':
+        is_second_reporting = True
+        reporting_interval = float(config['reporting_interval'][:-1])
+        reportingConfigVars['reporting_interval'] = float(reporting_interval / 60.0)
+    else:
+        reportingConfigVars['reporting_interval'] = int(config['reporting_interval'])
```

Comment:
Please keep the consistency of naming convention (either all camel case or name with underscore). The Camel case looks better to me